TestProjects/ShaderGraph/Assets/CommonAssets/Editor/ShaderGraphUITestHelpers.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using UnityEditor;
+using UnityEditor.ShaderGraph.UnitTests;
+using UnityEngine;
+using UnityEngine.UIElements;
+using UnityEngine.UIElements.StyleSheets;
+using UIECursor = UnityEngine.UIElements.Cursor;
+
+namespace UnityEditor.ShaderGraph.UnitTests
+{
+    static class ShaderGraphUITestHelpers
+    {
+        private static EventBase CreateEvent(Event evt)
+        {
+            Assembly uiElementAssembly = null;
+
+            foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
+            {
+                var assemblyName = assembly.GetName().ToString();
+                if (assemblyName.Contains("UnityEngine.UIElementsModule"))
+                {
+                    uiElementAssembly = assembly;
+                }
+            }
+            Type uiElementsUtilityType = uiElementAssembly?.GetType("UnityEngine.UIElements.UIElementsUtility");
+
+            MethodInfo createEventMethodInfo = uiElementsUtilityType?.GetMethod("CreateEvent",
```

Comment:
Also, cache the MethodInfo in a class member the first time you grab it so you only need to reflect once and can then just invoke it for subsequent events.

Reply:
Will do! Good call 👍